pootle/apps/pootle_store/models.py

```diff
@@ -1342,6 +1358,13 @@ class Store(models.Model, CachedTreeItem, base.TranslationStore):
                                             related_name='stores',
                                             db_index=True, editable=False)
 
+    filetype = models.ForeignKey(
+        Format,
+        related_name='stores',
+        null=True, blank=True,
+        db_index=True)
+    is_template = models.BooleanField(default=False)
```

Comment:
i disagree.

This PR creates the db fields necessary to remove `localfiletype` this is one of them